### ISSUE TITLE
marquez-web:0.43 does not appear to work on kubernetes

### ISSUE THREAD

**Issue by davidsharp7** [MarquezProject/marquez#2709]
When firing up marquez-web:0.43.0 on kubernetes it returns 

```bash
illegal instruction (core dumped)
```

then moves into crash loop back off.  No other information is presented back.

marquez-web:0.42.0 worked without issue.  

Checked with @phixMe and no significant changes have been made but I did notice that the image layers looked very different to previous builds.

**Comment by davidsharp7** [comment]
Docker manifests appear different

0.42.0

```json
[
    {
        "Id": "sha256:8d32f8c73853f1256f7850d60c77ea952a4a15d67417809dbfaace7ee413af1c",
        "RepoTags": [
            "marquezproject/marquez-web:0.42.0"
        ],
        "RepoDigests": [
            "marquezproject/marquez-web@sha256:5a38b77516d81d9e1e45f626b5a6d9909323335abf9b37c2bc15d6d06a615171"
        ],
        "Parent": "",
        "Comment": "",
        "Created": "2023-10-17T13:14:32.340321362Z",
        "Container": "7bd0e977f1156b9a2a9a8e24517bdfab782dc816c5422ccbcf97e6b299cb43aa",
        "ContainerConfig": {
            "Hostname": "7bd0e977f115",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "3000/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "NODE_VERSION=18.18.2",
                "YARN_VERSION=1.22.19"
            ],
            "Cmd": [
                "/bin/sh",
                "-c",
                "#(nop) ",
                "ENTRYPOINT [\"/usr/src/app/entrypoint.sh\"]"
            ],
            "Image": "sha256:0815606fd72918e8a2e48ed2c0cbdafafb759b2bdc48f5ee1ef8565937e14c6c",
            "Volumes": null,
            "WorkingDir": "/usr/src/app",
            "Entrypoint": [
                "/usr/src/app/entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": {}
        },
        "DockerVersion": "20.10.24",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "3000/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "NODE_VERSION=18.18.2",
                "YARN_VERSION=1.22.19"
            ],
            "Cmd": null,
            "Image": "sha256:0815606fd72918e8a2e48ed2c0cbdafafb759b2bdc48f5ee1ef8565937e14c6c",
            "Volumes": null,
            "WorkingDir": "/usr/src/app",
            "Entrypoint": [
                "/usr/src/app/entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": null
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 652519496,
        "VirtualSize": 652519496,
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/1295ce155016b8c2b133cd0e456e0bee21d236e006a05b7f544e36a3ca65ba6f/diff:/var/lib/docker/overlay2/6d5b72390797fc827ef35e3beecc374a6933f612d1fe8f1a05d454cb33ec2051/diff:/var/lib/docker/overlay2/ea18294298419f4431f30d38bb55be2b2eaf5921343c3ca3cb4bc3b00d7be665/diff:/var/lib/docker/overlay2/d2958fd3d665fee8567962ea013c94c7bea10afae855bb84d8e57568c8f18c7a/diff:/var/lib/docker/overlay2/bdc99f988c26ac44ea61f59e5e37e31e4c112841b1d581546d45a4fdbe43001a/diff:/var/lib/docker/overlay2/5a6e2fa645124479c7d86de8420d52d4e78594d7df322198c005b4da2039bc0c/diff:/var/lib/docker/overlay2/e9d9934d9aa299c6f800f242775e9efcc027aa26b8d45d4027cbf4782fd0d0fe/diff:/var/lib/docker/overlay2/35a854d9b9c3b47e28a7bf82255b944efb7b948d7bb0d82dec6f3ee4ed6bc173/diff:/var/lib/docker/overlay2/cdfb57209cf9591eb24e6d4132e15937ac4a3fde864d518f1c8d3bac4e1a8209/diff:/var/lib/docker/overlay2/dbf97e43b0a57e80c7bee496fcb42d89325efc8f8559488e7109347ee5c20d9f/diff:/var/lib/docker/overlay2/a984cd00f629add1e1ebbca635dfc9911d84a6cc53f783f84c81e949b10ae54f/diff",
                "MergedDir": "/var/lib/docker/overlay2/58759547c3acbd03d8d8a5a9185967740c824767025c072b72165b79bf5efd4d/merged",
                "UpperDir": "/var/lib/docker/overlay2/58759547c3acbd03d8d8a5a9185967740c824767025c072b72165b79bf5efd4d/diff",
                "WorkDir": "/var/lib/docker/overlay2/58759547c3acbd03d8d8a5a9185967740c824767025c072b72165b79bf5efd4d/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:cc2447e1835a40530975ab80bb1f872fbab0f2a0faecf2ab16fbbb89b3589438",
                "sha256:b4d059375c501a7c27bb978f73880b97c529647f8172e8e1d0798afd9feda732",
                "sha256:f845afce9f520cf26f019ca48e7f842af34d5a40197844707512d26c0c5a5b84",
                "sha256:ba0c1f87d7a9b66d76281c01863b0e0778d0f5729a686cbd69a343d3b1d33331",
                "sha256:fb526e2986fe13c8473ddd11c473023ec274e890838cc4e349af9abffd66c09f",
                "sha256:4362060400bb9e608e362d50bcca58a956ade79fb29e2016bd711b628ba72cfa",
                "sha256:cea0166776ea66b55ac0b8799487de7e19c0a1647eda63da59574bbb3064c6d8",
                "sha256:a9ec20cec98d341d1a52eace2bc49393722db8bdc6e95507d6e62e121156b927",
                "sha256:48e21abd1e2e92e28c30c6eed42ca7756a6cfda799645b9c523ab7de7d6e5415",
                "sha256:d090dfeef5491ac62c3070fe9a8501ca9eedc24fc3d44e4c1621a493dab38f69",
                "sha256:43f0635f34ce779a85ba872187ae27a3cd33f388972412149b10742ae0a536ce",
                "sha256:9cd015d66a7a85b87d5a48e7de4075038951b8ee05c4eb65982b66991277064b"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]
```

0.43.0
```json
[
    {
        "Id": "sha256:71290c4b83833bebc18c58c29743073e62847af1d40e0a29d1b04195c04ede86",
        "RepoTags": [
            "marquezproject/marquez-web:0.43.0"
        ],
        "RepoDigests": [
            "marquezproject/marquez-web@sha256:6b823f0ff1b9aa780ca7e2711105c4d52542a14ea7ccdc63682f5112e06f1dc2"
        ],
        "Parent": "",
        "Comment": "buildkit.dockerfile.v0",
        "Created": "2023-12-15T19:38:18.77537035Z",
        "Container": "",
        "ContainerConfig": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": null,
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": null
        },
        "DockerVersion": "",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "3000/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "NODE_VERSION=18.19.0",
                "YARN_VERSION=1.22.19"
            ],
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "/usr/src/app",
            "Entrypoint": [
                "/usr/src/app/entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": null
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 607932977,
        "VirtualSize": 607932977,
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/2faf72a8dab84457500983c63e130e98775f24361ba5bd23d755cca7140404e4/diff:/var/lib/docker/overlay2/151037bcbafaf0c1cac2280a6c4c412c4168e1f79d9bead61b279285d89e3a6e/diff:/var/lib/docker/overlay2/c0df7b2534377f8cbeb6db0556d03307944f651afbd31517a829096f5784b597/diff:/var/lib/docker/overlay2/4856c687e6387edfed7e30f150f79babcf0f9e315ef2e958b096a7c6a9493d7a/diff:/var/lib/docker/overlay2/98085f7c682f020cfbc5a6cce97725436f8a794d54826f191a5742c9f68a13c2/diff:/var/lib/docker/overlay2/c92bc2a34a7be0d28ab557353d6cb1ca16a5ccab31d61fb8ffa57af56221cfb5/diff:/var/lib/docker/overlay2/2895e334c4691ca30e112cc86ed3b8425ac509118cf647285f38497eac672681/diff:/var/lib/docker/overlay2/1168f46b7c64ccfd481621659ad463ccee2b42949fc00307c96cc4357261916c/diff:/var/lib/docker/overlay2/a5a13e9a99e536f9fb586a49236461b868cb8586b7c9829c16aece449daf36f4/diff:/var/lib/docker/overlay2/12ce2dd571ae2c1e63f6d4377edb2dda1f9d9d843f0b2ffbf8504716981cb6a7/diff:/var/lib/docker/overlay2/3e5f150de342c7cb08250202cc10f8f50adfd2b46e345b0f0235d1cfe6323dcd/diff",
                "MergedDir": "/var/lib/docker/overlay2/025e4bc0450f0604c1a4b3d41ab9db509eb87b1ba5caf03021842a10a70b1844/merged",
                "UpperDir": "/var/lib/docker/overlay2/025e4bc0450f0604c1a4b3d41ab9db509eb87b1ba5caf03021842a10a70b1844/diff",
                "WorkDir": "/var/lib/docker/overlay2/025e4bc0450f0604c1a4b3d41ab9db509eb87b1ba5caf03021842a10a70b1844/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:5af4f8f59b764c64c6def53f52ada809fe38d528441d08d01c206dfb3fc3b691",
                "sha256:62bb8375428b10bf660febe0b9819175d19de272f45ddbbc5db8db9426913256",
                "sha256:aec6514001fe82bb9a720e47e50208562f8db245910ba9ff862441dd7f846654",
                "sha256:0ae13a02eeea43822675d6efa7784f49c7a1c472882d2ea98b9e5ba9dbc494d4",
                "sha256:51e0db8e1027c4b6aac6006378a1178982aa5f15f2ca5c419de4c7aff48a6104",
                "sha256:d8c7384ca487f3b1430ac5e3ad378d52e18503070a9a548552b517d13ae75805",
                "sha256:9ecd3f7ec211edf30cef449640bdb11648d855a2524dc458ed8bfe667e8ca465",
                "sha256:cc247a7235d03361bd0e3597117aa04040142af4cd216ad56c06a0694ca33b29",
                "sha256:4cad66e868cb5f750cabb42a7b79eca7c4c960e3433aed06807e3dbe5bbd9170",
                "sha256:6452669f84cc2c52939c6274c1d6f5c7e75e6bf8993e60f33b6335f4350a5d5e",
                "sha256:ba71183ef0ca9c0828e7eceaedc7735dd30a0805c68b0e653a331feadf04ed1f",
                "sha256:e740c20c57c672efb0ee4688cb756130b49788f6dedd74df2492da2d6eda140c"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]
```

I've run a debug container on kubernetes and can start the web server manually so I think its a architecture issue per se.

**Comment by davidsharp7** [comment]
Closing as this seems to be resolved with version 0.44.0
